### ISSUE TITLE
Configurable Contact-Method Throttling

### ISSUE THREAD

**Issue by jonathanspw** [target/goalert#2778]
**What problem would you like to solve? Please describe:**
Add the ability to customize contact-method throttling which was implemented in https://github.com/target/goalert/releases/tag/v0.27.0

**Describe the solution you'd like:**
It would be great if the CM throttling added in 0.27.0 were customizable.  Here is my use-case:

We have several heavy sleepers that take several phone calls to wake up.  We need to call them multiple times before giving up and falling back to the escalation policy.  They setup their alerts for instant SMS alert and phone call, with a 1m delayed phone call, 2m delayed phone call, 3m delayed phone call, etc.

On a busy night where they might get called for multiple incidents this setup will immediately run into rate limits and cause the escalation policy to be followed when it otherwise wouldn't have, and potential unnecessary delays in fixing issues.

TL;DR sometimes we need to spam call people several times in a row :)

**Describe alternatives you've considered:**
I can't think of any other way to accomplish the goal here.

**Comment by Lakshmi2107** [comment]
Might be linked to #2653
